src/loc/en-us.ts

```diff
@@ -354,6 +354,10 @@ define([], () => {
     CollectionDataItemMissingFields: "Fields collection is empty!",
     InvalidUrlError: "The provided URL is not valid",
     CollectionDataSearch: "Search",
+MyTeamsLoadingMessage: "loading you teams",
+MyTeamsMessageError: "Something went wrong while loading your teams, please try later or refresh browser",
+
```

Comment:
Is MyTeamsMessageDontHaveTeams missing here?

Reply:
done